src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
```

Comment:
- [ ] Kindly use redux toolkit instead of redux to follow the project requirements :blush:

Reply:
Implemented. Thank you.